masterlist.yaml

```diff
@@ -12970,6 +12975,16 @@ plugins:
         crc: 0x50EEF62E
         util: '[TES4Edit v4.0.3](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 138
+    clean:
+      - crc: 0x489D7BE6
+        util: 'TES4Edit v4.0.4c'
+  - name: 'Kvatch Rebuilt - No More Burned Ground.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/15412/'
+        name: 'Kvatch Rebuilt'
+    req: [ 'Kvatch Rebuilt.esp' ]
+    msg:
+      - <<: *optional
```

Comment:
```suggestion
```

This plugin already has `Kvatch Rebuilt.esp` as a master, so a requirement is redundant. `optional` is also only for optional plugins that are packaged with the main plugin & that mod managers will therefore install by default, informing users that they may not actually need or want it. That leaves the location which isn't useful on its own, so the plugin can be removed (unless you want to add other metadata for it).

Reply:
Replaced with the dirty data for the plugin. Considering that it's an additional download from Kvatch Rebuilt rather than its own mod, I think it's probably helpful to have the link in there for the no more burned ground esp.